modules/oadp-backing-up-pvs-csi.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+[id="oadp-backing-up-pvs-csi_{context}"]
+= Backing up persistent volumes with Container Snapshot Interface snapshots
+
+You back up persistent volumes with Container Snapshot Interface (CSI) snapshots by creating a `VolumeSnapshotClass` CR.
+
+.Prerequisites
+
+* The cloud provider must support CSI snapshots.
+* You must install the OpenShift API for Data Protection (OADP) Operator and the Data Protection Application.
+* You must add the `csi` default plugin and the `EnableCSI` feature flag to the `DataProtectionApplication` manifest:
```

Comment:
s/`DataProtectionApplication` manifest/`DataProtectionApplication` CR ??

Reply:
A manifest is not the same thing as a CR.

Reply:
So the the `DataProtectionApplication` CR. and the `DataProtectionApplication` manifest are two separate, distinct objects? Same with `Backup` CR and `Backup` manifest. .